apis/iam/v1beta1/diff.go

```diff
@@ -102,10 +99,8 @@ func compareAuditConfigs(desired, actual []IAMPolicyAuditConfig) *structuredrepo
 	canonicalize := func(configs []IAMPolicyAuditConfig) []IAMPolicyAuditConfig {
 		// Create a deep copy to avoid modifying the original struct.
```

Comment:
Using `slices.Clone` from the standard library (if Go version allows) would be a more idiomatic way to copy the slice. If not, the current implementation is fine.